### DISCUSSION TITLE
Question about the hyperparameters BATCH_EXAMPLE

### DISCUSSION THREAD

**Discussion by Oceanusity** [ACEsuit/mace#167] [Q&A] [answered]
Hello authors, thanks for providing this wonderful works. I am currently running the MACE code, and try to understand the symmetric_contraction part. After running the code, I am a little confused about the hyper parameters [BATCH_EXAMPLE](https://github.com/ACEsuit/mace/blob/42659c3aa84f3318e568343261b4a8635fce0166/mace/modules/symmetric_contraction.py#L19). I guess it is not related to the batch size of the graphs. Would you mind providing some guidance about this? Thanks.

**Comment by davkovacs** [comment, marked as answer]
Hi! That is just used for the automatic optimisation of the einsum operations, for details see: https://github.com/Linux-cpp-lisp/opt_einsum_fx
